src/Algebra/Graph.agda

```diff
@@ -21,7 +21,6 @@ data _≡_ {A} : (x y : Graph A) -> Set where
 
     -- Congruence
     +left-congruence  : ∀ {x y z : Graph A} -> x ≡ y -> x + z ≡ y + z
-    +right-congruence : ∀ {x y z : Graph A} -> x ≡ y -> z + x ≡ z + y
```

Comment:
Can you leave a comment here, that `+right-congruence` actually does hold, but is a theorem? Also, since the proof is pretty short and doesn't really need the proof syntax sugar, I suggest you move it this file.